web_apis.fnc

```diff
@@ -4141,7 +4146,7 @@ Content-Type: application/atom+xml
 	END IF;
 
 	IF t_outputType = 'html' THEN
-		t_output :=                                                                                                                                                                                                                          
+		t_output :=
```

Comment:
My editor was set to auto-delete trailing spaces on lines, which it did in two places above. They didn't look significant, but flagging in case they are.

Reply:
Thanks for your attention to detail.  Neither of those are significant.